src/persist-client/src/cache.rs

```diff
@@ -197,6 +197,7 @@ async fn blob_rtt_latency_task(
             match blob.get(BLOB_GET_LIVENESS_KEY).await {
                 Ok(_) => {}
                 Err(_) => {
+                    next_measurement = tokio::time::Instant::now() + measurement_interval;
```

Comment:
at this point it seems simpler to rearrange so the gauge is only set in the Ok path and the rest of the bookkeeping is shared:

```
if let Ok(_) = blob.get(BLOB_GET_LIVENESS_KEY).await {
    metrics.blob.rtt_latency.set(start.elapsed().as_secs_f64());
}
next_measurement = tokio::time::Instant::now() + measurement_interval;
```

Reply:
oh yeah, that's definitely better, but I don't want to eat another CI cycle

Reply:
typed this up as a followup PR, which I'll open once this one merges